vocabularies/Analytics.xml

```diff
@@ -176,6 +179,129 @@
+        </Parameter>
+        <Parameter Name="ShowLevels" Type="Edm.Int64" Nullable="false">
+          <Annotation Term="Core.Description" String="Number of levels to be shown in the initial expansion (absent means all levels)" />
+          <Annotation Term="Core.OptionalParameter">
+            <Record/>
+          </Annotation>
+        </Parameter>
+        <Parameter Name="ExpandEntries" Type="Collection(Analytics.MultiLevelExpandEntry)" Nullable="false">
+          <Annotation Term="Core.Description" String="Entries with exceptional expansion" />
+          <Annotation Term="Core.OptionalParameter">
+            <Record/>
+          </Annotation>
+        </Parameter>
+        <Parameter Name="SubtotalsAtBottom" Type="Edm.Bool" Nullable="false">
+          <Annotation Term="Core.Description" String="Whether to duplicate the group headers so that they appear before and after their descendants" />
+          <Annotation Term="Core.LongDescription">
+            <String>The entry before has [DrillState](Hierarchy.md#HierarchyType) `expanded`,
+            the entry after has DrillState `subtotal`.</String>
+          </Annotation>
+          <Annotation Term="Core.OptionalParameter">
+            <Record>
+              <PropertyValue Property="DefaultValue" String="false" />
+            </Record>
+          </Annotation>
+        </Parameter>
+        <ReturnType Type="Collection(Edm.EntityType)">
+          <Annotation Term="Core.Description" String="Output set including the instance annotation [`LevelInformation`](#LevelInformation)" />
+        </ReturnType>
+      </Function>
+      <ComplexType Name="MultiLevelExpandLevel">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="Property names constituting a level in an [unnamed leveled hierarchy](#MultiLevelExpand)" />
+        <Annotation Term="Core.LongDescription">
+          <String>`DimensionProperties` must be used to identify entries in [`ExpandEntries/Entry`](#MultiLevelExpandEntry),
+          otherwise they have the same effect as `AdditionalProperties`.</String>
+        </Annotation>
+        <Property Name="DimensionProperties" Type="Collection(Edm.String)" Nullable="false">
+          <Annotation Term="Core.Description" String="A non-empty set of property names that define a combination of dimension values" />
+        </Property>
+        <Property Name="AdditionalProperties" Type="Collection(Edm.String)" Nullable="false">
+          <Annotation Term="Core.Description" String="A possibly empty set of names of additional properties of the dimensions that occur in `DimensionProperties`" />
+        </Property>
+      </ComplexType>
+      <ComplexType Name="MultiLevelExpandSiblingOrder">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="Sibling order in an [unnamed leveled hierarchy](#MultiLevelExpand)" />
```

Comment:
Hmm, this is not at all specific to hierarchies, multi-level, or unnamed.

Reply:
I don't understand. The term "sibling" is defined only in a hierarchy.

Reply:
OK, "sibling". But apart from that, it would apply to any ordinary collection as well.

Reply:
Do what?

Reply:
As I tried to say: let's treat this as "case closed", but I don't know how to tell that to this tool.